README.md

```diff
@@ -161,7 +161,7 @@ SIG-specific GitHub discussions.
 | <a id="sig-collector"></a>Collector | Alternating between Tuesday at 17:00 PT, Wednesday at 09:00 PT, and Wednesday at 05:00 PT | [Google Doc](https://docs.google.com/document/d/1r2JC5MB7GupCE7N32EwGEXs9V_YIsPgoFiLP4VWVMkE) | [#otel-collector](https://cloud-native.slack.com/archives/C01N6P7KR6W) | [calendar-collector](https://groups.google.com/a/opentelemetry.io/g/calendar-collector) | [Pablo Baeyens](https://github.com/mx-psi) |
 | <a id="sig-cpp"></a>C++: SDK | Alternating between Monday at 13:00 PT and Wednesday at 09:00 PT | [Google Doc](https://docs.google.com/document/d/1i1E4-_y4uJ083lCutKGDhkpi3n4_e774SBLi9hPLocw) | [#otel-cpp](https://cloud-native.slack.com/archives/C01N3AT62SJ) | [calendar-cpp](https://groups.google.com/a/opentelemetry.io/g/calendar-cpp) | [Severin Neumann](https://github.com/svrnm) |
 | <a id="sig-dotnet-auto-instr"></a>.NET: Automatic Instrumentation | Wednesday at 09:00 PT | [Google Doc](https://docs.google.com/document/d/1dYdwRQVE3zu0vlp_lqGctNm0dCQUkDo2LfScUJzpuT8) | [#otel-dotnet-auto-instr](https://cloud-native.slack.com/archives/C01NR1YLSE7) | [calendar-dotnet](https://groups.google.com/a/opentelemetry.io/g/calendar-dotnet) | [Morgan McLean](https://github.com/mtwo) |
-| <a id="sig-dotnet"></a>.NET: SDK | Tuesday alternating between 09:00 PT and 16:00 PT | [Google Doc](https://docs.google.com/document/d/1yjjD6aBcLxlRazYrawukDgrhZMObwHARJbB9glWdHj8) | [#otel-dotnet](https://cloud-native.slack.com/archives/C01N3BC2W7Q) | [calendar-dotnet](https://groups.google.com/a/opentelemetry.io/g/calendar-dotnet) | [Morgan McLean](https://github.com/mtwo) |
+| <a id="sig-dotnet"></a>.NET: SDK | Tuesday at 11:00 PT | [Google Doc](https://docs.google.com/document/d/1yjjD6aBcLxlRazYrawukDgrhZMObwHARJbB9glWdHj8) | [#otel-dotnet](https://cloud-native.slack.com/archives/C01N3BC2W7Q) | [calendar-dotnet](https://groups.google.com/a/opentelemetry.io/g/calendar-dotnet) | [Morgan McLean](https://github.com/mtwo) |
```

Comment:
You will need to update https://github.com/open-telemetry/community/blob/main/sigs.yml instead and regenerate this file